BeeSwift/GoalViewController.swift

```diff
@@ -284,7 +287,70 @@ class GoalViewController: UIViewController,  UIScrollViewDelegate, DatapointTabl
         setValueTextField()
         updateInterfaceToMatchGoal()
     }
-
+    
+    enum MenuAction {
+        case goalCommitment
+        case goalStop
+        case goalData
+        case goalStatistics
+        case goalSettings
+        
+        func makeLink(username: String, goalName: String) -> URL? {
+            guard
+                let accessToken = ServiceLocator.currentUserManager.accessToken
+            else { return nil }
+            
+            let destinationUrl: URL
+            
+            switch self {
+            case .goalCommitment:
+                destinationUrl = DeeplinkGenerator.generateDeepLinkToGoalCommitment(username: username, goalName: goalName)
+            case .goalStop:
+                destinationUrl = DeeplinkGenerator.generateDeepLinkToGoalStop(username: username, goalName: goalName)
+            case .goalData:
+                destinationUrl = DeeplinkGenerator.generateDeepLinkToGoalData(username: username, goalName: goalName)
+            case .goalStatistics:
+                destinationUrl = DeeplinkGenerator.generateDeepLinkToGoalStatistics(username: username, goalName: goalName)
+            case .goalSettings:
+                destinationUrl = DeeplinkGenerator.generateDeepLinkToGoalSettings(username: username, goalName: goalName)
+            }
+            
+            return DeeplinkGenerator.generateDeepLinkToUrl(accessToken: accessToken, username: username, url: destinationUrl)
+        }
+    }
+    
+    struct MenuOption {
+        let title: String
+        let action: MenuAction
+        let imageSystemName: String
+    }
+    
+    private func getMenuOptions() -> [MenuOption] {
+        [
+            MenuOption(title: "Commitment", action: .goalCommitment, imageSystemName: "signature"),
+            MenuOption(title: "Stop/Pause", action: .goalStop, imageSystemName: "pause.fill"),
```

Comment:
With this icon the button ends up looking more like an action (i.e. clicking it will stop/pause the goal) rather than a link to a page. Any ideas for an alternative?

Reply:
A few options come to mind:
- No icons
- all use the sf symbol safari since they all open in the browser
- introduce [TipKit](https://developer.apple.com/documentation/tipkit/highlightingappfeatureswithtipkit) to suggest using the feature (explaining a tad what they do)
- leave it as is
- use other icons
- reword title of menu, possibly of entries as well

Reply:
I've thought about this some more, and decided:
1) I don't immediately have an idea I love
2) If/when people are confused about this, they'll click it, and learn what it actually does without harm.